bin/replication.js

```diff
@@ -2,10 +2,14 @@ const async = require('async');
 const program = require('commander');
 const { S3, IAM, SharedIniFileCredentials } = require('aws-sdk');
 
-const { Logger } = require('werelogs');
-
+const werelogs = require('werelogs');
+const { RoundRobin } = require('arsenal').network;
+const Logger = werelogs.Logger;
```

Comment:
Minor: You might move this below the requires. Alternatively, couldn't we keep `const { Logger } = require('werelogs');`?